org.opent2t.onboarding.getverahub/js/thingOnboarding.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+var https = require('https');
+var cryptojs_sha1 = require('crypto-js');
+var request = require('request-promise');
+
+class Onboarding {
```

Comment:
yes, i'll get rid of that once we pull the existing vera translators which use it so it's all clean.

---
In reply to: [103618235](https://github.com/openT2T/onboarding/pull/32#discussion_r103618235) [](ancestors = 103618235)